.github/workflows/run-tests.yml

```diff
@@ -46,4 +46,16 @@ jobs:
         run: |
           npm run check
           npm run test
+          while true; do \
+            MATCH="$(sudo netstat -tlnp | grep 27018 || true)"; \
+            if [[ "$MATCH" == "" ]]; then echo "mongod exited..."; break; fi; \
+            echo "$MATCH"; \
+            echo "mongod still up..."; \
+            PID="$(echo $MATCH | awk '{print $7}' | cut -f1 -d'/')"; \
+            if [[ "$PID" == "" ]]; then echo "failed to find PID"; break; fi; \
+            echo "sudo killing -9 $PID..."; \
+            sudo kill -9 $PID; \
+            echo "... killed?!"; \
```

Comment:
Maybe we don't need this here anymore?

Reply:
I've explicitly left it in there to always know whether that command completed or not 😅